simtools/model/site_model.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/python3
+
+import logging
+
+from simtools.model.model_parameter import ModelParameter
+
+__all__ = ["SiteModel"]
+
+
+class SiteModel(ModelParameter):
+    """
+    SiteModel represents the MC model of an observatory site.
+
+    Parameters
+    ----------
+    site: str
+        Site name (e.g., South or North).
+    mongo_db_config: dict
+        MongoDB configuration.
+    model_version: str
+        Version of the model (ex. prod5).
+    label: str
+        Instance label. Important for output file naming.
+    """
+
+    def __init__(
+        self,
+        site,
+        mongo_db_config=None,
+        model_version="Released",
+        db=None,
+        label=None,
+    ):
+        """
+        Initialize SiteModel
+        """
+        self._logger = logging.getLogger(__name__)
+        self._logger.debug("Init SiteModel for site %s", site)
+        ModelParameter.__init__(
+            self,
+            site=site,
+            mongo_db_config=mongo_db_config,
+            model_version=model_version,
+            db=db,
+            label=label,
+        )
+
+    def get_reference_point(self):
+        """
+        Get reference point coordinates as dict
+
+        Returns
+        -------
+        dict
+            Reference point coordinates as dict
+        """
+
+        return {
+            "center_altitude": self.get_parameter_value_with_unit("reference_point_altitude"),
+            "center_northing": self.get_parameter_value_with_unit("reference_point_utm_north"),
+            "center_easting": self.get_parameter_value_with_unit("reference_point_utm_east"),
+            "epsg_code": self.get_parameter_value("epsg_code"),
+        }
+
+    def get_corsika_site_parameters(self, config_file_style=False):
+        """
+        Get site-related CORSIKA parameters as dict.
+        Parameters are returned with units wherever possible.
+
+        Parameters
+        ----------
+        config_file_style bool
+            Return using style of corsika_parameters.yml file
+
+        Returns
+        -------
+        dict
+            Site-related CORSIKA parameters as dict
+
+        """
+
+        # backwards compatibility to `corsika_parameters.yml` (temporary)
```

Comment:
Added a TODO.

Agree the first line is a bit weird but required to allow for similar outputs as in the past. Good to have a TODO there now, so that we remember to remove it future.